src/app/page.tsx

```
@@ -0,0 +1,37 @@
+//   height: 100vh;
+//   position: sticky;
+//   top: 0;
+//   background-color: #d8e3ff; //테스트용 나중에 배경색 바꾸기
+//   display: flex;
+//   justify-content: center;
+//   align-items: center;
+// `;
+
+export default function () {
+  return <div></div>;
+}
```

ESLint

[error] Component definition is missing display name

(react/display-name)